File: in_Python/282A_Bit++.py
num = int(input());
initial = 0

for i in range(num):
    op = str(input())
    
    if op[0] == '-' or op[2] == '-' or op[2] == '-' :
        initial = initial - 1
    elif op[0] == '+' or op[2] == '+' or op[2] == '+' :
        initial = initial + 1 
print(initial)    

File: in_C++/282A_Bit++.cpp
#include <iostream>

using namespace std;
int main()
{
    int num;
    cin >> num;

    int initial = 0;

    for (int i = 0; i < num; i++)
    {
        string op;
        cin >> op;
        
        if (op[0] == '+' || op[1] == '+' || op[2] == '+')
        {
            initial += 1;
        }
        else if (op[0] == '-' || op[1] == '-' || op[2] == '-')
        {
            initial -= 1;
        }
    }
    cout << initial << endl;
}